stores/sql/sqlite/migrations/main/migration_00027_contract_usability.sql

```diff
@@ -0,0 +1,2 @@
+ALTER TABLE contracts ADD COLUMN usability INTEGER NOT NULL DEFAULT 0;
```

Comment:
Considered not defaulting and recreating the contracts table with `bad` contracts. Follow up migration that removes contract sets would then mark all contracts in the set as good for renew.